src/org/rust/lang/intentions/ExpandModule.kt

```diff
@@ -8,15 +8,18 @@ import com.intellij.psi.PsiFile
 import com.intellij.psi.impl.file.PsiFileImplUtil
 import com.intellij.refactoring.move.moveFilesOrDirectories.MoveFilesOrDirectoriesUtil
 import org.rust.lang.core.psi.impl.RustFileImpl
+import org.rust.lang.core.psi.util.LIB_RS
+import org.rust.lang.core.psi.util.MAIN_RS
 import org.rust.lang.core.psi.util.MOD_RS
 
 class ExpandModule : IntentionAction {
+    private val DIRECTORY_OWNER_FILE_NAMES = setOf(MOD_RS, MAIN_RS, LIB_RS)
 
     override fun getFamilyName() = "Expand module structure"
     override fun startInWriteAction() = true
     override fun getText() = "Expand Module"
     override fun isAvailable(project: Project, editor: Editor?, file: PsiFile?) =
-        file is RustFileImpl && file.name != MOD_RS
+        file is RustFileImpl && file.name !in DIRECTORY_OWNER_FILE_NAMES
```

Comment:
It is better use `https://github.com/alexeykudinkin/intellij-rust/pull/150/files#diff-221e21326806663a21f37048add38cd3R27`.

Reasons: DRY and logic for determining if a module is a dir owner **will** change